### ISSUE TITLE
Blog tag filter list incomplete

### ISSUE THREAD

**Issue by natestemen** [unitaryfund/unitary.fund#517]
The list of dropdown tags on [`/blog`](https://unitary.fund/blog/) are incomplete. Currently what's displayed is the following, where `qulacs` and `workshop` are cut off.
<img width="300" alt="Screenshot 2024-08-05 at 12 17 40 PM" src="https://github.com/user-attachments/assets/3f54a0af-b09a-419f-ae5b-abc2d3c9f796">

We have many more tags on blog posts, however. Typing into the box does not bring up missing tags, either.

**Comment by nathanshammah** [comment]
I'm working on this, closing also #349.

**Comment by natestemen** [comment]
Is the goal then to get to <17[^1] tags? Just curious since I was not thinking about/testing that scenario when reviewing #515. If so, this hypothesis should be relatively straightforward to test by removing all but two blog posts with distinct sets of tags and spinning up the website locally.

Even if this feature works with <17 total tags, it's still broken (IMO) and requires fixing.

[^1]: The number of tags that are currently displayed on https://unitary.fund.

**Comment by nathanshammah** [comment]
Ping for @KallieFerguson and @beecastanon who are working on a solution.

**Comment by willzeng** [comment]
I believe this is now fixed. If not, we can reopen

**Comment by beecastanon** [comment]
@willzeng This should be fixed, yes. Thanks for closing.

**Comment by natestemen** [comment]
Just did a quick test to ensure we are not limited in the # of tags we can use on the blog by adding additional unique tags to a blog post locally. Sure enough they all show up in the local build!

Thanks @KallieFerguson for cleaning these up in https://github.com/unitaryfund/unitary.fund/pull/595! There must have been a syntax error or other that you cleaned up which fixed this issue while cleaning everything up.

I found one small bug which I cleaned up in https://github.com/unitaryfund/unitary.fund/commit/a2b919021bdbe14a17ac7523bce72f50dae7bc0f.
